etc/kayobe/kolla/config/prometheus/haproxy.rules

```diff
@@ -6,17 +6,8 @@ groups:
 - name: HAProxy
   rules:
 
-  - alert: HaproxyDown
-    expr: haproxy_up == 0
```

Comment:
cAdvisor should spot a container down - we no longer have this metric because the exporter is built it.